Bucketeer/Sources/Internal/Scheduler/EvaluationForegroundTask.swift

```diff
@@ -61,8 +59,11 @@ final class EvaluationForegroundTask: ScheduledTask {
                         seconds: response.seconds,
                         sizeByte: response.sizeByte
                     )
-                    // reset retry count
-                    self?.retryCount = 0
+                    // reset the scheduler to use the default polling interval configured in the BKTConfig
+                    if (retryCount > 0) {
+                        self?.retryCount = 0
+                        self?.reschedule(interval: pollingInterval)
```

Comment:
Fixes the polling interval setting not being reset when the request succeeds after retrying.